Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -254,26 +254,28 @@ public void SearchValidationsRecursive(DependencyObject depObj)
 
         private void UpdatePrevNextButton()
         {
-
-            if (mWizard.IsLastPage())
+            if (mWizard.IsFirstPage())
             {
-                xNextButton.IsEnabled = false;
-                xFinishButton.IsEnabled = true;
+                xPrevButton.IsEnabled = false;
             }
             else
             {
-                xNextButton.IsEnabled = true;
+                xPrevButton.IsEnabled = true;
             }
 
-            if (mWizard.IsFirstPage())
+            if (mWizard.IsLastPage())
             {
-                xPrevButton.IsEnabled = false;
+                xNextButton.IsEnabled = false;
+                xFinishButton.IsEnabled = true;
+                if (mWizard.DisableBackBtnOnLastPage == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4375598)